backend/typescript/services/implementations/creatorService.ts

```diff
@@ -58,6 +58,19 @@ class CreatorService implements ICreatorService {
       throw error;
     }
   }
+
+  async approveCreator(userId: string): Promise<void> {
+    try {
+      const creator = await this.getCreatorById(userId);
+      creator.isApproved = true;
```

Comment:
This is hackey and I'm not sure if the change will propagate to the database. If there is a better way to do this (setters & getters for example) then I would opt to use that method instead.

Reply:
Yeah I'm pretty sure this isn't how our ORM works. Looking at the docs for Sequelize, using a model instance is probably your best bet, use the findByPk method to get the instance from ID then update + save. 

Test it with the following steps:
```
sudo docker exec -it ccbc-db-1 bash
psql -U postgres -d ccbc
select * from users;
```
make sure the users table reflects your changes

Reply:
https://sequelize.org/docs/v6/core-concepts/model-instances/

take a look at userService.ts implementation around line 229